index.html

```diff
@@ -27,7 +27,115 @@ <h1>Charlie Giorgania</h1>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="grid-item">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this Project</button>
+        </div>
+        <div class="abtme">
+          <h1>About</h1>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ea eiusmod
+          </p>
+          <button>Get my cv</button>
+        </div>
+      </div>
+    </section>
+    <div class="card1">
+      <h2>Languages</h2>
+      <ul>
+        <li>Javascript(ES5/ES6)</li>
+        <li>RUBY</li>
+        <li>HTML</li>
+        <li>CSS</li>
+      </ul>
+      <img alt="" />
+    </div>
+    <div class="card2">
+      <h2>Frameworks</h2>
+      <ul>
+        <li>Bootstrap</li>
+        <li>Ruby on rails</li>
+        <li>RSpec</li>
+        <li>Capibara</li>
+        <li>Selenium</li>
+      </ul>
+      <img alt="" />
+    </div>
+    <div class="card3">
+      <h2>Skills</h2>
+      <ul>
+        <li>Codekit</li>
+        <li>Codepen</li>
+        <li>GitHub</li>
+        <li>GitLAb</li>
+        <li>Terminal</li>
+      </ul>
+      <img alt="" />
+    </div>
```

Comment:
- After moving the about section out of the works section, kidding move all these cards into the about section to improve your design layout and update your styles to make your design close as possible to the original design. this is what I mean below:

```
    <section class="about-me">
      <div class="abtme">
        <h1>About</h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
          ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
          aliquip ea eiusmod
        </p>
        <button>Get my cv</button>
      </div>
      <div class="card1">
        <h2>Languages</h2>
        <ul>
          <li>Javascript(ES5/ES6)</li>
          <li>RUBY</li>
          <li>HTML</li>
          <li>CSS</li>
        </ul>
        <img alt="" />
      </div>
      <div class="card2">
        <h2>Frameworks</h2>
        <ul>
          <li>Bootstrap</li>
          <li>Ruby on rails</li>
          <li>RSpec</li>
          <li>Capibara</li>
          <li>Selenium</li>
        </ul>
        <img alt="" />
      </div>
      <div class="card3">
        <h2>Skills</h2>
        <ul>
          <li>Codekit</li>
          <li>Codepen</li>
          <li>GitHub</li>
          <li>GitLAb</li>
          <li>Terminal</li>
        </ul>
        <img alt="" />
      </div>
  
    </section>

```